crates/ruma-common/src/events/room/create.rs

```diff
@@ -50,16 +56,58 @@ pub struct RoomCreateEventContent {
 }
 
 impl RoomCreateEventContent {
-    /// Creates a new `RoomCreateEventContent` with the given creator.
-    pub fn new(creator: OwnedUserId) -> Self {
+    /// Creates a new `RoomCreateEventContent` with the given creator, as required for room versions
+    /// 1 through 10.
+    pub fn new_v1(creator: OwnedUserId) -> Self {
+        #[allow(deprecated)]
         Self {
-            creator,
+            creator: Some(creator),
             federate: true,
             room_version: default_room_version_id(),
             predecessor: None,
             room_type: None,
         }
     }
+
+    /// Creates a new `RoomCreateEventContent` with the default values and no creator, as introduced
+    /// in room version 11.
+    ///
+    /// The room version defaults to [`RoomVersionId::V11`].
```

Comment:
"defaults to" sounds like there is an optional argument for it. I think this should probably just be

```suggestion
    /// The room version is set to [`RoomVersionId::V11`].
```